### ISSUE TITLE
`persistenceId` misisng in JournalTypeConverter.unmarshal signature

### ISSUE THREAD

**Issue by t3hnar** [dnvriend/akka-persistence-jdbc#22]
I'm using `persistenceId` to decide whether to use `binary` or `json` serialiser, however it is a bit problematic to deserialize with current `JournalTypeConverter.unmarshal` api.

**Comment by dnvriend** [comment]
Hi,

 I have released v1.1.6 that has the following signature for the `JournalTypeConverter.unmarshal`; it breaks the serialization API but now you can determine which serializer to use based on the persistenceId that will be passed to the API. 

```scala
def unmarshal(value: String, persistenceId: String)(implicit serialization: Serialization): PersistentRepr
```

Let me know what you think and have fun!

**Comment by t3hnar** [comment]
Thx @dnvriend!
